pages.ca/linux/apt-file.md

```diff
@@ -0,0 +1,16 @@
+# apt-file
```

Comment:
The example `--regexp` is missing, you translated from the Spanish page, but the Spanish page is outdated. We based all the translations in the main «English» pages. You can check the translation status (and outdated) here: https://lukwebsforge.github.io/tldri18n/
Could you please sync this translation with the English version?